metadata-models/src/main/resources/entity-registry.yml

```diff
@@ -297,4 +297,15 @@ entities:
       - queryProperties
       - querySubjects
       - status
+  - name: schema
+    category: core
+    keyAspect: schemaKey
+    aspects:
+      - ownership
+      - status
+      - domain
+      - subTypes
+      - schemaMetadata
+      - editableSchemaMetadata
```

Comment:
Should `schemaMetadata` aspects in `Dataset` entity be removed and add there just a reference (schema urn)?

Reply:
Maybe, instead of removed we may differentiate two possible scenarios: embedded schema within dataset schema and linked/referenced schema. 🤔